src/app/shared/entry/recent-events.pipe.ts

```diff
@@ -25,27 +24,15 @@ export class RecentEventsPipe implements PipeTransform {
 
     switch (type) {
       case 'displayName': {
-        if (event.workflow) {
-          return this.entryToDisplayNamePipe.transform(event.workflow);
-        } else if (event.tool) {
-          return this.entryToDisplayNamePipe.transform(event.tool);
-        } else if (event.apptool) {
-          return this.entryToDisplayNamePipe.transform(event.apptool);
-        }
-        break;
+        const entry = this.getEntry(event);
+        return entry && this.entryToDisplayNamePipe.transform(entry);
```

Comment:
Admittedly, the `&&` syntax is terse, but more compact than the equivalent `if`.